docs/blocks/custom-fluids.md

```diff
@@ -0,0 +1,851 @@
+            "condition": "query.block_neighbor_has_any_tag(1, 0, 0, 'template_full') || query.block_neighbor_has_any_tag(0, 0, 1, 'template_full') || query.block_neighbor_has_any_tag(-1, 0, 0, 'template_full') || query.block_neighbor_has_any_tag(0, 0, -1, 'template_full')",
+            "run_command": {
+              "target": "self",
+              "command": [ "execute if block ~ ~ ~1 air unless block ~ ~-1 ~ fluids:fluid_template_down run setblock ~ ~ ~1 fluids:fluid_template2" ]
+            }
+          },
+          {
+            "condition": "query.block_neighbor_has_any_tag(1, 0, 0, 'template_full') || query.block_neighbor_has_any_tag(0, 0, 1, 'template_full') || query.block_neighbor_has_any_tag(-1, 0, 0, 'template_full') || query.block_neighbor_has_any_tag(0, 0, -1, 'template_full')",
+            "run_command": {
+              "target": "self",
+              "command": [ "execute if block ~ ~ ~-1 air unless block ~ ~-1 ~ fluids:fluid_template_down run setblock ~ ~ ~-1 fluids:fluid_template2" ]
+            }
+          },
+          {
+            "condition": "query.block_neighbor_has_any_tag(1, 0, 0, 'template_full') || query.block_neighbor_has_any_tag(0, 0, 1, 'template_full') || query.block_neighbor_has_any_tag(-1, 0, 0, 'template_full') || query.block_neighbor_has_any_tag(0, 0, -1, 'template_full')",
+            "run_command": {
+              "target": "self",
+              "command": [ "execute if block ~1 ~ ~ air unless block ~ ~-1 ~ fluids:fluid_template_down run setblock ~1 ~ ~ fluids:fluid_template2" ]
+            }
+          },
+          {
+            "condition": "query.block_neighbor_has_any_tag(1, 0, 0, 'template_full') || query.block_neighbor_has_any_tag(0, 0, 1, 'template_full') || query.block_neighbor_has_any_tag(-1, 0, 0, 'template_full') || query.block_neighbor_has_any_tag(0, 0, -1, 'template_full')",
+            "run_command": {
+              "target": "self",
+              "command": [ "execute if block ~-1 ~ ~ air unless block ~ ~-1 ~ fluids:fluid_template_down run setblock ~-1 ~ ~ fluids:fluid_template2" ]
+            }
+          },
+
+          {
+            "condition": "!query.block_neighbor_has_any_tag(1, 0, 0, 'template_full') && !query.block_neighbor_has_any_tag(0, 0, 1, 'template_full') && !query.block_neighbor_has_any_tag(-1, 0, 0, 'template_full') && !query.block_neighbor_has_any_tag(0, 0, -1, 'template_full')",
+            "run_command": {
+              "target": "self",
+              "command": [ "setblock ~ ~ ~ air" ]
+            }
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+</Spoiler>
+
+## Outer Fluid Block 2
+
+Below is the JSON for the `outer fluid block 2`. To replicate the json, copy and quick replace `template` with your fluid's name. This block has the same function as the `outer fluid block 2` but it places `outer fluid block 3` instead.
+
+<CodeHeader>BH\blocks\fluid_template\fluid_template2.json</CodeHeader>
+<Spoiler title="Outer Fluid 2 JSON">
+
+```json
```

Comment:
Anyway we can use the latest `1.19.60` format version? Also shorten all query to q